src/containers/Nodes/getNodesColumns.tsx

```diff
@@ -69,7 +70,7 @@ const dataCenterColumn: NodesColumn = {
     name: NODES_COLUMNS_IDS.DC,
     header: 'DC',
     align: DataTable.LEFT,
-    render: ({row}) => (row.DataCenter ? row.DataCenter : '—'),
+    render: ({row}) => row.DC || EMPTY_DATA_PLACEHOLDER,
```

Comment:
In raw data we have `DataCenter` field. However, in code we reference it as `DC`, also backend sort works only with field short name. So replaced `DataCenter` field with `DC` everywhere, use `DataCenter` from node `Location` field, where it's possible (it's not present in `/viewer/json/compute` endpoint)